docs/pages/deploy-a-cluster/helm-deployments/aws.mdx

```diff
@@ -262,6 +262,15 @@ $ kubectl --namespace teleport create -f aws-issuer.yaml
 ## Step 4b/7. Configure Teleport to use ACM to handle TLS
 
 <Admonition type="warning" title="Warning">
+  
```

Comment:
I think we can organize this Admonition a lot better. Right now, it seems like a collection of miscellaneous notes. 

The new text in this Admonition is more of a mitigation step that applies after you have already set up Teleport with ACM. Can we add that new text to the next step instead? I think a `Details` box would make sense for this.